### ISSUE TITLE
Learn from <virtual-scoller>

### ISSUE THREAD

**Issue by buschtoens** [html-next/vertical-collection#245]
I just stumbled upon [`<virtual-scoller>`](https://github.com/valdrinkoshi/virtual-scroller). Maybe there are some ideas we can ~~steal~~ _improve upon_. 😄

**Comment by pzuraq** [comment]
Yeah absolutely, I did some research back when I first started working on this, but will definitely do it again once we start work on v2 😄

**Comment by pzuraq** [comment]
I took a look at this again today, didn't realize the first time that this was an actual spec! Definitely think that this being a built in primitive in HTML is the best possible place to handle this type of thing, there are just things we can do there that we won't be able to do in Javascript. 100% behind this proposal.

**Comment by runspired** [comment]
@pzuraq it's not a spec and it's not as far along as vertical-collection in many ways, makes design choices that are regrettable. Does have involvement from a few folks that occasionally do spec work though so perhaps they are hoping to learn enough to implement one: but given the limited nature of their case studies and the past history of bad specs many of them have proposed I'm skeptical.
